### PR TITLE
#61. Card text issues

### PR DESCRIPTION
**Pull request by fadilmamedov**

### CODE REVIEW

File: src/components/feed/item/styles.module.css [hunk 1]
```diff
@@ -1,4 +1,6 @@
 .wrapper {
+  display: flex;
```
Comment (shcheklein) [inline src/components/feed/item/styles.module.css]: why do we need this? could you elaborate please?

**Comment by shcheklein** [comment]
@fadilmamedov it looks like it breaks the main block on the large screen:

Screen Shot 2020-01-09 at 11 53 11 AM

File: src/components/feed/item/styles.module.css (hunk 1 above)
Comment (fadilmamedov) [reply src/components/feed/item/styles.module.css]: I wanted to put meta section at the very bottom of the card, which is possible by making a card container to be displayed as `flex` and meta section with `margin-top` auto. Keeping meta section absolutely positioned causes text overlapping

**Comment by fadilmamedov** [comment]
> @fadilmamedov it looks like it breaks the main block on the large screen:
> 
> Screen Shot 2020-01-09 at 11 53 11 AM

Hmmm... I can't reproduce this issue neither in Chrome nor in Firefox. In what browser were you able to reproduce this?

**Comment by fadilmamedov** [comment]
> @fadilmamedov it looks like it breaks the main block on the large screen:
> 
> Screen Shot 2020-01-09 at 11 53 11 AM

Fixed! Thanks!

File: src/components/feed/item/styles.module.css (hunk 1 above)
Comment (shcheklein) [reply src/components/feed/item/styles.module.css]: right! just curios - why was that happening on the medium screen size only?

**Comment by iAdramelk** [comment]
@fadilmamedov thanks for the pull request. Couple small bugs from me:

1. In the large block if there is too much content, fade is not shown:

image

2. Second issue is still present on iPhone 5:

image

3. Margin auto didn't move author info down in Firefox (and IE11, but we can live with that one):

image

**Comment by fadilmamedov** [comment]
@iAdramelk thanks a lot for your comments! I have fixed the first and the third issues you pointed out. But I am not sure about the second point. Do you think we need to take into account this issue for only iPhone 5/SE users? Because moving information about the read time will look weird. What do you think @shcheklein?

**Comment by shcheklein** [comment]
@fadilmamedov moving to the next line looks weird indeed, but it's not nice that it's broken still on certain layouts. Are there any other options, ideas?

**Comment by fadilmamedov** [comment]
@shcheklein ok, I have an idea. What about using a short date string, something like "Sep 05, 2019", if the row could not be placed in one single row?

**Comment by shcheklein** [comment]
@fadilmamedov yep, it sounds reasonable to me. Probably it should always keep the same format though - no matter if it fits or not. It would be simpler to implement.

**Comment by iAdramelk** [comment]
Thanks for the fixes! I'm ok with shorter date format.

**Comment by shcheklein** [comment]
@fadilmamedov any updates on this one?

**Comment by shcheklein** [comment]
closing as stale

**Pull request closed without merging**